### ISSUE TITLE
TODO for MVP

### ISSUE THREAD

**Issue by zopieux** [Zopieux/django-bugz#1]
* [ ] Expose all state in detail view (assignee, blocked by, dupe of)
* [ ] GitHub-like search
* [ ] Edit ticket: title, description, assignee, dupe, blocked by, ~labels~
* [ ] More tests
* [ ] Reverse-links (duplicate of → duplicated by) in event log
* [ ] Collapse rapid events of same type into one:
  > zopieux added labels a, b on … 12:10:01
  > zopieux removed label b on … 12:10:03
  > zopieux added label c on … 12:10:04
  ↓
  > foo added labels a, c on 12:10:01
* [ ] Decent CSS
